addons/html_editor/static/src/main/media/image_plugin.js

```diff
@@ -370,6 +371,8 @@ export class ImagePlugin extends Plugin {
     closeImageTransformation() {
         if (this.isImageTransformationOpen()) {
             registry.category("main_components").remove("ImageTransformation");
+            this.dispatch("CONTENT_UPDATED");
+            this.document.activeElement.blur();
```

Comment:
Why not target same behavior as in 17.0 where using transform on image removes both selection as well as the toolbar?